src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js

```diff
@@ -333,16 +330,18 @@ describe('ReactDOMServerIntegration', () => {
     });
 
     if (ReactDOMFeatureFlags.useFiber) {
-      itRenders('a array type children as a child', async render => {
-        class AppComponent extends React.Component {
```

Comment:
hmm. Has been resolved, it can tests passed both server and client. There is no warning.